### PR TITLE
Use decompiler

### PR DESCRIPTION
**Pull request by LoiNguyenCS**
Professor,

This is the PR to invoke a decompiler. I have decided to use [VineFlower](https://vineflower.org). It is built based on [FernFlower](https://github.com/fesh0r/fernflower). However, VineFlower supports Gradle, and it seems to be [cooler](https://www.reddit.com/r/java/comments/ue8u59/new_open_source_java_decompiler/) according to its creator. :-)

Sorry for adding the try-catch statements when parsing the file. As explained in the comments, it is actually difficult to fix a bug that can't be reproduced locally. But since those bugs do not happen locally., I think we don't have to worry about them.

### CODE REVIEW

File: src/test/java/org/checkerframework/specimin/JarFileTest.java
Comment (LoiNguyenCS) [inline src/test/java/org/checkerframework/specimin/JarFileTest.java:11]: I figured out this mistake while trying to run the test.

File: src/main/java/org/checkerframework/specimin/SpeciminRunner.java
Comment (kelloggm) [inline src/main/java/org/checkerframework/specimin/SpeciminRunner.java:208]: This does make me very nervous, but I agree it's okay to keep it for now. Please make this comment a TODO, and add another TODO at each of the other places that you've added such a catch below.

**Review by kelloggm — APPROVED** [review]
LGTM other than the need for a few more TODO comments. Can be merged without another review after those are added.

**Pull request merged**
